backend/src/routes/account/rest_api/account_controller.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import HTTPException,status,APIRouter
+from ..account_service import AccountService
+from pydantic import BaseModel
+class AccountDetails(BaseModel):
+    username: str
+    password: str
+
+router=APIRouter()
+
+@router.post("/login")
+async def login(account_details:AccountDetails):
+    account = await AccountService.login_user(account_details.username, account_details.password)
+    if not account:
+        raise HTTPException(status_code=status.HTTP_401_UNAUTHORIZED, detail="Incorrect username or password")
+    return {"message":"login successful"}
```

Comment:
let's send a success message with appropriate status code ( 200 ) and the account document that we are getting from AccountService.login_user